STYLEGUIDE.md

```diff
@@ -0,0 +1,215 @@
+# Particle Cloud Framework Style Guide
+
+This is a Python Style Guide for the PCF Framework. The intent of this guide is to strive for cleaner and better code quality. Following this guide, our code will be approachable to anyone who needs to maintain it later, including ourselves!
+
+## Table of Contents
+1. [Virtual Environments](#venv)
+2. [Imports](#imports)
+3. [Naming](#naming)  
+    i. [Variables](#variables)  
+    ii. [Class names](#classes)  
+    iii. [Functions/Methods](#funcs)
+4. [Spacing](#spacing)
+5. [Strings](#strings)  
+    i. [Quotes](#quotes)  
+    ii. [Docstrings](#docstr)
+6. [Dictionary Key Value Retrieval](#dict)
+7. [PCF Utility Functions](#pcf_util)
+8. [PCF Exceptions](#pcf_excep)
+
+## <a name="venv">Virtual Environments</a>
```

Comment:
I would add language around the idea that Conda is the preferred setup for testing and development, but you can use venv alone if you prefer. Since Python 3.3, venv has been a native python library,  making it is a lot easier for some to get rolling with. 
https://docs.python.org/3/library/venv.html